packages/fluent/sass.config.js

```diff
@@ -6,11 +6,11 @@ module.exports = {
         {
             file: 'scss/all.scss',
             outFile: 'dist/all.css',
-            compiler: 'sass-embedded',
+            compiler: 'sass',
```

Comment:
why?

Reply:
The CI build fails, as the `sass-embedded` os dependencies are options, thus running `npm ci --no-optional` causes an issue. Also, we have actually used `sass` up till now, as the root config has this inconsistency: [see here](https://github.com/telerik/kendo-themes/blob/develop/sass-swatches.config.js#L22-L27).

Thus, I am eliminating the issue, as well as standardizing on the used approach. Also, `sass-embedded` is now [part of the primary `sass` distributable](https://github.com/sass/dart-sass/releases/tag/1.63.0).